eclipse-license-resource-bundle/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2012-2018 Red Hat, Inc.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License v1.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-v10.html
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.eclipse.che.parent</groupId>
+    <artifactId>che-eclipse-license-resource-bundle</artifactId>
+    <version>6.8.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+    <name>Eclipse license codestyle</name>
+    <description>Header Texts and others License resources</description>
+
+    <!-- NOTE: this file is not source validated for sort / license stuff, because it's upstream from the parent pom -->
```

Comment:
This is the only consequence of having this pom NOT depend on the parent pom. It can't self-check itself for sort order and license accuracy. 

But by doing this, we break the parent-depends-on-license-bundle and license-bundle-depends-on-parent thing.